php/php.editor/test/unit/data/testfiles/structure/php53/bracketed_multiple_namespaces_with_multiple_default.php

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Please remove this and add the license header.

Reply:
Done